jsconfig.json

```diff
@@ -2,7 +2,12 @@
     "compilerOptions": {
         "target": "esnext",
         "module": "commonjs",
-        "baseUrl": "src"
+        "baseUrl": "src",
+        "paths": {
+            "~/*": [
+                "*"
+            ]
+        }
```

Comment:
I hated having to do imports with lots of ../../../../ so this allows you to also use paths relative to the src/ folder like
```
import useApi from "hooks/useApi";
import testApi from "api/test";
```
See examples of imports below if ur confused